constants/index.js

```diff
@@ -23,7 +23,7 @@ import {
   TOTALS,
   TRUE_STRING,
 } from './vars'
-import { THEME, FontSize14InRem } from './theme'
+import { THEME, fontSize } from './theme'
```

Comment:
The export pattern is all uppercase

Reply:
I see a mix of cases in the constants file.

![Screenshot 2023-11-08 at 2 54 09 PM](https://github.com/AMP-SCZ/dpdash/assets/2905145/b6bbd589-9edc-4b5b-8f68-3ad67a9e56e6)

Reply:
👍